pyproject.toml

```diff
@@ -56,9 +57,31 @@ python -m ipykernel install --user --name "conda_braket" --display-name "conda_b
 pytest --dist worksteal -n logical -ra -v --durations=0 test/
 """
 
+[tool.hatch.envs.lint]
+detached = true
+dependencies = [
+  "black[jupyter]>=23.1.0",
+]
+
+[tool.hatch.envs.lint.scripts]
+style = [
+  "black --check --diff {args:.}",
+]
+fmt = [
```

Comment:
Did you run the style check on the repo and it didn't recommend any changes?

Reply:
I had ran this and there were suggestions which can be fixed with ` hatch run lint:fmt  `. However, for introducing this, I was adding this in a shadow run mode for the time being without the changes.